pkg/apis/logging/v1/types.go

```diff
@@ -1,4 +1,4 @@
-package v1alpha1
+package v1
 
 import (
 	"github.com/openshift/elasticsearch-operator/pkg/apis/elasticsearch/v1alpha1"
```

Comment:
```
	v1 "k8s.io/api/core/v1"
```
do you need to change this import to `corev1` as in `zz_generated.deepcopy.go`?

Reply:
only if there is a conflict and compile error.  That file is generated so they have taken the liberty of using a naming convention